File: spring_java_orm_sample/src/main/java/com/example/demo/openbd/config/OpenBdProperties.java
package com.example.demo.openbd.config;

import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;
import org.springframework.validation.annotation.Validated;

@Component
@ConfigurationProperties(prefix = "openbd")
@Validated
@Getter
@Setter
public class OpenBdProperties {
    private String baseUrl = "https://api.openbd.jp";
}

File: codex_poting/kotlin/src/main/kotlin/com/example/demo/openbd/config/OpenBdProperties.kt
package com.example.demo.openbd.config

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component
import org.springframework.validation.annotation.Validated

@Component
@ConfigurationProperties(prefix = "openbd")
@Validated
class OpenBdProperties {
    var baseUrl: String = "https://api.openbd.jp"
}
